### PR TITLE
Upgraded Rubocop to 0.40

### PR DESCRIPTION
**Pull request by alebian**
## Summary

Updated Rubocop version

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -122,7 +122,7 @@ group :development, :test do
   gem 'faker'
 
   # Lints
-  gem 'rubocop'
+  gem 'rubocop', '0.40.0'
```
Comment (epintos) [inline Gemfile]: Why leaving this static version only? Its better to use the `~` notation

Comment (epintos) [inline Gemfile]: @alebian

**Comment by alebian** [comment]
@epintos true :)

**Comment by epintos** [comment]
👍

**Pull request merged**